mmf/models/__init__.py

```diff
@@ -16,6 +16,7 @@
 from .unimodal import UnimodalBase, UnimodalText, UnimodalModal
 from .visual_bert import VisualBERT
 from .vilbert import ViLBERT
+from .albef.vit import AlbefVitEncoder
```

Comment:
We should only expose models here. Once the albef model is added you can add the import here.